joint_state_publisher_gui/package.xml

```diff
@@ -0,0 +1,23 @@
+<package format="2">
+  <name>joint_state_publisher_gui</name>
+  <version>1.12.13</version>
+  <description>
+    This package contains a GUI tool for setting and publishing joint state values for a given URDF.
+  </description>
+  <author email="davidvlu@gmail.com">David V. Lu!!</author>
+  <author email="jacquelinekay1@gmail.com">Jackie Kay</author>
+  <maintainer email="clalancette@osrfoundation.org">Chris Lalancette</maintainer>
+  <maintainer email="sloretz@osrfoundation.org">Shane Loretz</maintainer>
+
+  <license>BSD</license>
+  <url type="website">http://www.ros.org/wiki/joint_state_publisher</url>
+  <url type="repository">https://github.com/ros/joint_state_publisher</url>
+  <url type="bugtracker">https://github.com/ros/joint_state_publisher/issues</url>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <exec_depend>joint_state_publisher</exec_depend>
+  <exec_depend>rospy</exec_depend>
+  <exec_depend>python_qt_binding</exec_depend>
+  <exec_depend>sensor_msgs</exec_depend>
```

Comment:
What's the rationale for this ordering? I'm surprised it's not alphabetic, or rospy-first-then-alphabetic as it appears it used to be.

Reply:
I just failed at alphabetizing :).  Fixed in 1db983d.